src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -3231,60 +3231,64 @@ namespace ReportSizingManager {
                             CoilInEnth = PsyHFnTdbW(CoilInTemp, CoilInHumRat);
                             CoilInWetBulb = PsyTwbFnTdbWPb(CoilInTemp, CoilInHumRat, StdBaroPress, CallingRoutine);
                             CoilOutEnth = PsyHFnTdbW(CoilOutTemp, CoilOutHumRat);
-                            SupFanNum = PrimaryAirSystem(CurSysNum).SupFanNum;
-                            RetFanNum = PrimaryAirSystem(CurSysNum).RetFanNum;
-                            switch (PrimaryAirSystem(CurSysNum).supFanModelTypeEnum) {
-                            case DataAirSystems::structArrayLegacyFanModels: {
-                                FanCoolLoad = FanDesHeatGain(PrimaryAirSystem(CurSysNum).SupFanNum, DesVolFlow);
-                                if (coilSelectionReportObj->isCompTypeCoil(CompType) && (SupFanNum > 0)) {
-                                    coilSelectionReportObj->setCoilSupplyFanInfo(CompName,
-                                                                                 CompType,
-                                                                                 Fans::Fan(PrimaryAirSystem(CurSysNum).SupFanNum).FanName,
-                                                                                 DataAirSystems::structArrayLegacyFanModels,
-                                                                                 PrimaryAirSystem(CurSysNum).SupFanNum);
-                                }
+                            if ( CurOASysNum > 0 ) { // coil is in the OA stream
```

Comment:
Added placeholder for OA system fan heat. For now, fan heat is ignored for OA system cooling coils.